backend/python/vllm/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+MY_DIR="$(dirname -- "${BASH_SOURCE[0]}")"
+
+uv venv ${MY_DIR}/venv
+source ${MY_DIR}/venv/bin/activate
+
+uv pip install --requirement ${MY_DIR}/requirements.txt
+
+if [ -f "requirements-${BUILD_TYPE}.txt" ]; then
+    uv pip install --requirement ${MY_DIR}/requirements-${BUILD_TYPE}.txt
```

Comment:
This would need a cuda file to install flash-attn as it's not a vLLM direct dependency, but it benefits from it if it's installed.